modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/NumericInputMask/NumericInputMask.tsx

```diff
@@ -33,13 +33,16 @@ interface INumericInputMaskValue {
 	append?: string;
 	appendType?: 'prefix' | 'suffix';
 	decimalSymbol?: DecimalSymbol[];
+	symbols: LocalizedValue<ISymbols>;
 	thousandsSeparator?: ThousandsSeparator[];
 }
 interface IProps {
 	append?: string;
 	appendType?: 'prefix' | 'suffix';
 	decimalSymbol: DecimalSymbol[];
 	decimalSymbols: ISelectProps<DecimalSymbol>[];
+	defaultLanguageId: Locale;
+	editingLanguageId: Locale;
```

Comment:
Sorry I didn't figure out it before, but those properties are not being used so you can delete them  
```suggestion
```